utils/apiclient.ts

```diff
@@ -0,0 +1,7 @@
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
```

Comment:
can you move this file into clients/admin.ts ?

Reply:
Fixed in [5f377a4](https://github.com/pagopa/io-functions-services/pull/68/commits/5f377a49099c30bdca6dbaa2140ae01e448402f4)